samples/certs/Makefile

```diff
@@ -72,9 +78,15 @@ root-key.pem:
 ##<name>-certs:	generate Citadel certificates for <name>. Includes all PEM files needed.
 .PHONY: %-certs
 
+ifeq ($(WORKLOAD_CERTS),0)
 %-certs: %/cert-chain.pem root-cert.pem
 	@echo "Citadel inputs stored in $(dir $<)"
 	@cp root-cert.pem $(dir $<)
+else
+%-certs: %/cert-chain.pem root-cert.pem %/workload-cert-chain.pem
+	@echo "Citadel inputs and workload certs stored in $(dir $<)"
+	@cp root-cert.pem $(dir $<)
```

Comment:
A small improvement (removes dir) might be

@cp root-cert.pem $(?D)

Would have to verify.